exec/java-exec/src/main/codegen/templates/EventBasedRecordWriter.java

```diff
@@ -96,7 +96,7 @@ private void initFieldWriters() throws IOException {
 
   public static abstract class FieldConverter {
     protected static final String LIST = "list";
-    protected static final String ELEMENT = "element";
+    public static final String ELEMENT = "element";
```

Comment:
Please other two constants public for consistency.

Reply:
why ? They are not used outside of converter.

Reply:
Just for consistency, since `ELEMENT` is already used outside, others might be also used with time. You can leave as is, it was just a suggestion. Anyway, if you leave as is, please order constants according to access modifies: having protected / public / protected is incorrect order.

Reply:
I agree with ordering remark. Regarding visibility I'd like to follow YAGNI principle here.

Reply:
I think it is better to move the definition of `ELEMENT` and `LIST` to `ParquetRecordWriter` because `FieldConverter` is used for other (csv, json etc.) formats as well and the group names are needed for Parquet only and it's up to writer to "decide" how to write a file. What do you think?

Reply:
ok.

Reply:
Only thing here that the constants should be part of parent ```ParquetOutputRecordWriter```.

Reply:
@KazydubB , done.